reference/OpenAPI/differences.md

```diff
@@ -0,0 +1,80 @@
+# Differences from Swagger
+
+FLAT uses an adapted version Swagger.
+
+## `x-flat-…` Extensions
+
+First of all, several extensions named `x-flat-…` are recognized on different levels of the definition:
+
+* `x-flat-flow`: [flow](routing.md#assigning-flat-flows) to be started. Recognized at top-level and below `paths`, `paths/<path>` and `paths/<path>/<operation>`.
+* `x-flat-init`: [init flow](routing.md#init-flow) (top-level)
+* `x-flat-error`: [error flow](routing.md#error-flow) (top-level)
+* x-flat-cors: [CORS configuration](cors.md) (top-level)
+* x-flat-validate: [validation](validation.md) (top-level)
+
+## Slimline Definition
+
+Defining endpoints in `swagger.yaml` should be straight-forward. However, typing Swagger compliant YAML or JSON can be rather cumbersome. To assign a [flow](routing.md#assigning-flat-flows) to a path with two operations (`get` and `post`) the Swagger specification requires you to write at least:
+
+```yaml
+swagger: "2.0"
+info:
+  title: ""
+  version: "1"
+paths:
+  /login:
+    x-flat-flow: login-flow.xml
+    get:
+      responses:
+        default:
+          description: ""
+    post:
+      responses:
+        default:
+          description: ""
+```
+
+In FLAT you may omit the top-level `info` section. And as long as you don't need a schema for [response validation](validation.md), you may even drop the useless `responses` sections:
+
+```yaml
+swagger: "2.0"
+paths:
+  /login:
+    x-flat-flow: login-flow.xml
+    get:
+    post:
+```
+
+If you don't care which HTTP methods are allowed for a path, you may even go without the concrete operations `get` and `post`:
+
+```yaml
+swagger: "2.0"
+paths:
+  /login:
+    x-flat-flow: login-flow.xml
+```
+
+It is by no means a bad thing to write a conformant definition and provide a `description` or `version` information. But often it slows you unnecessarily down and worsens readability.
```

Comment:
slows you down unnecessarily?

Reply:
But it often...  oder However, it frequently ...

down unnecessarily and degrades readability